package.json

```diff
@@ -48,7 +48,7 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
     "babel-preset-stage-0": "^6.5.0",
-    "babel-register": "^6.11.5",
+    "babel-register": "6.11.5",
```

Comment:
Locking this in at 6.11.5, since the latest version of `babel-register` (6.11.6) is causing `npm test` to fail when sourceMaps are enabled.